link/src/main/java/com/stripe/android/link/LinkPaymentLauncher.kt

```diff
@@ -242,7 +242,7 @@ class LinkPaymentLauncher @AssistedInject internal constructor(
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     companion object {
-        val LINK_ENABLED = BuildConfig.DEBUG
+        val LINK_ENABLED = false
```

Comment:
Revert this change?

Reply:
Nope, had to do this to fix `PaymentOptionsActivityTest` for now.